test/stat_tracker_test.rb

```diff
@@ -22,6 +22,20 @@ def test_game_collection
     assert_instance_of GameCollection, @stat_tracker.game_collection
   end
 
+  def test_it_show_highest_total_score
+    assert_equal 7, @stat_tracker.highest_total_score
+  end
+
+  def test_it_show_lowest_total_score
+    assert_equal 3, @stat_tracker.lowest_total_score
+  end
+
+  def test_it_show_biggest_blowout
+    assert_equal 2, @stat_tracker.biggest_blowout
+  end
+
+
```

Comment:
I would recommend removing the extra lines between test_it_show_biggest_blowout and test_calculates_percentage_home_wins